Dockerfile

```diff
@@ -18,9 +18,7 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
-EXPOSE 8080
+EXPOSE 8081
```

Comment:
Ensure that the exposed port 8081 here matches the port configuration in the `docker-compose.yml` file to avoid connectivity issues.